xml/obs_request_and_review_system.xml

```diff
@@ -98,6 +98,12 @@
     <para> can be used to update the devel package information in the target.
     </para>
    </sect3>
+   <sect3>
+    <title><literal>release</literal></title>
+    <para> Is used to release a ready a build. Sources and binaries are copied
```

Comment:
The first sentence sounds strange. :wink:  Do you mean "Is used to release ready/complete/finished build"?

Reply:
yeah, clarified that